osr/waspobject.simba

```diff
@@ -833,10 +833,67 @@ begin
   end;
 end;
 
+procedure HoverMSTile(DotType: ERSMinimapDot; RightClick: Boolean = False);//Copied from Flight
+//Belongs in Minimap most likely
+//Modified for ASyncMouse
+var
+  Tries: Int32;
+  msBox: TBox;
+  tpa,cTPA: TPointArray;
+  cArr: TIntegerArray;
+begin
+  tpa := Minimap.GetDots(DotType);
+  if tpa.Len() < 1 then Exit;
+  repeat
+    inc(Tries);
+    msBox := Minimap.PointToMsBox(tpa[random(low(tpa),high(tpa))]);
+    if MainScreen.Bounds.Contains(msBox) then
+    begin
+      msBox := msBox.Expand(5);
+      if (not MainScreen.Bounds.Contains(msBox)) then Continue;
+      // Get all colors on the tile
+      // Extract the rarest
+      // Gather points
+      // Mouse to random point
+      cArr := getColors(tpaFromBox(msBox));
+      if (SRL.FindColors(cTPA, cArr.Min, msBox) > 0) then
+      begin
+        ASyncMouse.Move(cTPA.Mean());
+      end;
+      exit;
+    end;
+  until(Tries > 10);
+end;
+
+procedure MouseOffClient(direction: Byte);//Copied from old SRL, modified
```

Comment:
This is already sort of included in SRL with `procedure TAntiban.RandomMouse;`
I would prefer to call that instead even though it's not exactly the same.